packages/ui/src/TabbedPanel/TabbedPanel.tsx

```diff
@@ -4,9 +4,14 @@ import { getOrientation, onTabDown } from "./helper";
 
 import type { Properties } from "./types";
 
-const TabbedPanel: React.FC<Properties> = ({ children, position = "top" }) => {
+const TabbedPanel: React.FC<Properties> = ({
+  children,
+  defaultActiveIndex = 0,
+  onTabChange,
+  position = "top",
+}) => {
   const id = useId();
-  const [active, setActive] = useState(0);
+  const [active, setActive] = useState(defaultActiveIndex);
```

Comment:
should we also add an useEffect to make this reactive?
```suggestion
  const [active, setActive] = useState(defaultActiveIndex);
  useEffect(() => {setActive(defaultActiveIndex);}, [defaultActiveIndex])
```
this should allow for change of defaultActiveIndex programatically by dev depending on different use cases.